assignments/objects.js

```diff
@@ -18,26 +18,72 @@ const example = {
 }
 
 // Write your intern objects here:
+const Mitzi = {
```

Comment:
It is a convention to name variables using camel casing. Usually PascalCasing is used only for constructor functions. This code works nonetheless. Following conventions  will make documentations friendly, for instance if you look at the MDN docs you can easily guess that anything with a PascalCasing is a function (constructor function) and those with camelCasing are either methods or properties.